dashboard/app/views/pd/workshop_mailer/_pre_course_survey.html.haml

```diff
@@ -1,5 +1,5 @@
 %h3
-  Pre-Course Survey
+  Pre-Workshopg Survey
```

Comment:
Small typo^

Reply:
grrr... that g which shows up when I start typing git without leaving my editor. haha

Reply:
So relatable😅